### DISCUSSION TITLE
How to visualize your local db in VSCode?

### DISCUSSION THREAD

**Discussion by salieflewis** [ponder-sh/ponder#330] [Question]
I'd like to be able to visualize the local database that gets stored in `.ponder/cache.db`

I started exploring some extensions such as [this](https://marketplace.visualstudio.com/items?itemName=alexcvzz.vscode-sqlite) one, but was having some troubles with it out of the box.

It looks like the next step for troubleshooting is to install sqlite globally on your computer via `sudo apt install sqlite`. Instead I tried to do this with Homebrew `brew install sqlite`, but was running into conflicts thanks to the native version of sqlite that macOS ships with.

Curious if anyone has found a working solution for this/gone down this rabbithole before.

**Comment by typedarray** [comment]
Cool idea. One thing to note is that at the moment, directly querying the database isn't "supported", but I encourage you to play around with this because ultimately it would be great to expose the database to other tools.

Also, we're planning to add a web UI for debugging/exploring your app, which would include a database explorer like what you're describing.

**Reply by salieflewis** [reply]
Very exciting re: web app! Let me know if I can be of any help in that vein. 

And yes, will continue to play around with it and maybe discover something!
